types/index.ts

```diff
@@ -1,5 +1,5 @@
 export type PayButton = {
```

Comment:
I don't think this should be in this branch

Reply:
done

Reply:
I tried to mess with the position of brackets, parenthesis and spaces in a file with 4 spaces identation and then committing the changes; but they were committed unlinted, am I testing this correctly?
![image](https://user-images.githubusercontent.com/21281174/168585590-c4751a11-ba45-416b-93f0-8c5e65333cad.png)